src/mame/orca/vastar.cpp

```diff
@@ -309,6 +352,9 @@ void vastar_state::draw_sprites(bitmap_ind16 &bitmap, const rectangle &cliprect)
 //  for (int offs = 0; offs < 0x40; offs += 2)
 	for (int offs = 0x40 - 2; offs >= 0; offs -= 2)
 	{
+		if (!(offs & 0x10))
```

Comment:
This plus other mods in PR will cause regressions in the other games used by this driver.

Reply:
I think these changes are correct for the other games.  I haven't tried this PR, but it does look like there are 2 lists of 0x10 entries, with gaps between them.

I suspect Planet Probe might expect them to be drawn swapped (but still in order) too, I noted that when I worked on it some years ago.  Other changes could be correct for the rest of the games too, probably same video system with some lines swapped?